README.md

```diff
@@ -34,33 +29,27 @@ Following individual component can be configured within each Jenkins job
 
 ## [SCMViewer for Subversion](https://github.com/jenkinsci/collabnet-plugin/blob/master/docs/SCM.md)
 
-# Help/Support:
-
-Please submit any questions to the [TeamForge forum](http://forums.open.collab.net/ds/viewForumSummary.do?dsForumId=736 "TeamForge forum").
-
 # FAQ:
 
    1.  Does this integrate with / update the [SFEE Plugin](https://wiki.jenkins.io/display/JENKINS/SFEE+Plugin)?
 
-        No, the [SFEE Plugin](https://wiki.jenkins.io/display/JENKINS/SFEE+Plugin) is a separate plugin that is not maintained by [CollabNet](http://www.collab.net/ "CollabNet").  It works with CollabNet SourceForge 4.4, so it may be a good choice if you have an older version.  Our plugin is not a direct upgrade.  If you change from using the [SFEE Plugin](https://wiki.jenkins.io/display/JENKINS/SFEE+Plugin) to the CollabNet Plugin, you will have to reconfigure your jobs appropriately. You should be able to use both plugins simultaneously, if you wish, although it might be a bit confusing.
+        No, the [SFEE Plugin](https://wiki.jenkins.io/display/JENKINS/SFEE+Plugin) is a separate plugin that is not maintained by [Digital.ai](https://digital.ai/ "Digital.ai"). .
 
    2.  Uploads fail when using an SSL-enabled TeamForge server.  What do I do?
 
-        You'll need to add the server's certificate to your java keystore on the Jenkins server.  First dowload the certificate to your Jenkins server.  You should be able to get it by opening a browser window on the TeamForge server, viewing the certificate associated with that page, and exporting the certificate file.  Then, you'll need to run something like "sudo keytool -keystore $JAVA\_HOME/jre/lib/security/cacerts -import -file teamforge.cert".  (On Windows, replace $JAVA\_HOME with %JAVA\_HOME%).  If you haven't changed your java keystore's password, it will be "changeit".  After you've imported the cert to the java instance that Jenkins is using, it should be able to upload.
+        You'll need to add the server's certificate to your java keystore on the Jenkins server.  First dowload the certificate to your Jenkins server.            You should be able to get it by opening a browser window on the TeamForge server, viewing the certificate associated with that page, and          exporting the certificate file.  Then, you'll need to run something like "sudo keytool -keystore $JAVA\_HOME/jre/lib/security/cacerts -                   import -file teamforge.cert".  (On Windows, replace $JAVA\_HOME with %JAVA\_HOME%).  If you haven't changed your java keystore's                    password, it will be "changeit".  After you've imported the cert to the java instance that Jenkins is using, it should be able to upload.
 
-   3.  I'm having problems with login and logout when using CollabNet Authentication.  It seems to be redirecting to an unexpected site.
+   3.  I'm having problems with login and logout when using Digital.ai Authentication.   It seems to be redirecting to an unex pected site.
```

Comment:
Please remove the empty space in 'unex pected'

Reply:
Resolved the comment.